tensorflow/python/kernel_tests/split_op_test.py

```diff
@@ -153,6 +153,14 @@ def _testSpecialCasesVariable(self):
       self.assertAllEqual(result[0], inp[0:1, :])
       self.assertAllEqual(result[1], inp[1:4, :])
 
+      result = self.evaluate(array_ops.split(inp, [-1, 4], 0))
+      self.assertAllEqual(result[0], inp[0:0, :])
+      self.assertAllEqual(result[1], inp[0:4, :])
+
+      result = self.evaluate(array_ops.split(inp, [4, -1], 0))
```

Comment:
Maybe add a test to cover splitting with -1's in a non-first axis as well.

Reply:
Done.